RecoJets/JetProducers/python/MVAJetPuId_cfi.py

```diff
@@ -0,0 +1,28 @@
+from RecoJets.JetProducers.hltPUIdAlgo_cff import *
```

Comment:
Could you rename
MVAJetPuId_cfi.py
to
hltMVAJetPuId_cfi.py
Otherwise it may be confused with the offline ID configuration. And it is fully consistent with the naming of hltPUIdAlgo_cfi.py and hltPUIdParams_cfi.py